apps/src/code-studio/initApp/loadApp.js

```diff
@@ -407,6 +408,9 @@ function loadAppAsync(appOptions) {
           progress.showDisabledBubblesAlert();
         }
       }
+      if (data.isVerifiedTeacher) {
+        store.dispatch(setVerified());
+      }
```

Comment:
This will likely cause a merge conflict with my promise race PR, which I'll have to watch out for.